CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.0]
+### Changed
+- `RestRequestListener` will not try to resolve `RestRequestOptions` on `kernel.request` event after resolving route configuration.
```

Comment:
I guess it's in reverse? Maybe we should communicate what changes, not what's happening there? For example:

When using annotations, RestRequestOptions will be available right after kernel.request event instead of kernel.controller event. This allows to show proper REST errors when exceptions are raised in the firewall